modules/contact/include/utils/MortarContactUtils.h

```diff
@@ -0,0 +1,116 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+#pragma once
+
+#include "FEProblemBase.h"
+
+#include "timpi/parallel_sync.h"
+#include "timpi/communicator.h"
+
+namespace Moose
+{
+namespace Contact
+{
+namespace Mortar
```

Comment:
I think I would prefer that `Mortar` be outside of `Contact` since we have `Moose::Mortar` invocations in the framework